docs-ref-conceptual/toc.yml

```diff
@@ -122,6 +122,8 @@
       items:
         - name: Overview
           href: java-azure-tools.md
+        - name: Supported Azure JDKs
```

Comment:
I prefer to call this "Supported Java JDKs". We do not have an "Azure JDK".

Reply:
The document should also say that customers can run other JDKs, but commercial support is provided automatically if they use Azul's